doc/user/content/release-notes.md

```diff
@@ -21,12 +21,28 @@ the PR description instead. They will be migrated here during the release
 process by the release notes team.
 {{< /comment >}}
 
-{{% version-header v0.26.0 %}}
+{{% version-header v0.26.1 %}}
 
-v0.26.0 is a **long-term support (LTS)** release of Materialize. We will support
```

Comment:
Should we maybe say that the `0.26.x` series of versions are LTS, and only the latest of them is supported at any point in time?

Reply:
I guess my point here is clarifying that the 0.26.x family of releases is what's being supported in the long term, not each individual 0.26 release. Nitpick, anyway!

Reply:
I agree with Alessandro

Reply:
Tweaked it slightly, added that bit as a note and got a 👍 from @umanwizard.